src/env.h

```diff
@@ -165,6 +165,7 @@ namespace node {
   V(priority_string, "priority")                                              \
   V(processed_string, "processed")                                            \
   V(prototype_string, "prototype")                                            \
+  V(public_key_string, "publicKey")                                           \
```

Comment:
Actually, the name of this field is Subject Public Key Info which contains not only a public key but also a key algorithm.